Gruntfile.js

```diff
@@ -133,42 +133,30 @@ module.exports = function( grunt ) {
 
 					"requirejs/require.js": "requirejs/require.js",
 					"requirejs-domready/domReady.js": "requirejs-domready/domReady.js",
-					"requirejs-text/text.js": "requirejs-text/text.js",
+					"requirejs-text/text.js": "requirejs-text/text.js"
 				}
 			}
 		},
-		jshint: {
+		eslint: {
 			options: {
-				jshintrc: true
+
+				// See https://github.com/sindresorhus/grunt-eslint/issues/119
+				quiet: true
 			},
-			all: {
-				src: [ files.source, files.grunt, files.karma, files.speed, files.tests ]
-			}
-		},
-		jscs: {
 			src: {
-				options: {
-					requireDotNotation: null
-				},
-				src: [ files.source ]
+				src: files.source
+			},
+			dev: {
+				src: [ files.source, files.grunt, files.karma, files.speed, files.tests ]
```

Comment:
Why is `files.source` listed twice, in `src` & `dev` sections?

Reply:
Please, check the new configuration. I have tried to do the same as jQuery